### ISSUE TITLE
Test the cifar100 dataset with low natural accuracy and robust accuracy

### ISSUE THREAD

**Issue by momo1986** [LayneH/self-adaptive-training#6]
Hi, 

I use the training procedure to run the self-adaptive-training, however, during testing, the accuracy and robust_accuracy go down to about 0.2.

What I have done to pgd_attack.py is only change the load dataset of cifar-10 to cifar-100, the model is setting with 100-classes output.

I use the default run_sat.sh setting to train the adversarial defense model.

What is the root cause for the low accuracy, how to resolve it?

Thanks & Regards!
Momo

**Comment by LayneH** [comment]
Hi,

Please note that the input range of `main_adv.py` and `pgd_attack.py` is [0, 1], which is different from `main.py`.

It seems that you used the `scripts/cifar100/run_sat.sh` for *standard training* to train the model and use `pgd_attack.py` to evaluate the model. Am I right? In such a case, I suggest you modify `scripts/cifar10/run_trades_sat.sh` for cifar100 and use `main_adv.py`  for adv training.

**Comment by AiXia520** [comment]
Hi, 
I run a clean CIFAR100 with the self-adaptive-training. I use the default SelfAdaptiveTrainingCE Loss. However, as the epoch increased, the training loss dropped first and then rose, and the learning rate was set by default in the self-adaptive-training. 
What is the root cause for this phenomenon, how to resolve it?

**Comment by LayneH** [comment]
Hi @AiXia520,

The loss increases since the training targets are smoother at each epoch (by the EMA update). However, this phenomenon does not affect the performance of SAT. And you may calibrate it by sharpening the logits, e.g., temperature scaling on the logits.
